en/docs/develop/product-apis/devportal-apis/devportal-v1/devportal-v1.yaml

```
@@ -5236,6 +5252,14 @@ components:
       required: true
       schema:
         type: string
+    includeCommenterInfo:
+      name: includeCommenterInfo
+      in: query
+      description: |
+        Whether we need to display commentor details.
```

```suggestion
        Specifies whether to include commentor details.
```
Or use a better discription

---

https://github.com/wso2/docs-apim/pull/2477/commits/d6a36d4b997ad585b9c66431a8e4e748a0314836